var/spack/repos/builtin/packages/py-line-profiler/package.py

```diff
@@ -11,18 +11,29 @@
 class PyLineProfiler(PythonPackage):
     """Line-by-line profiler."""
 
-    homepage = "https://github.com/rkern/line_profiler"
+    homepage = "https://github.com/pyutils/line_profiler"
     pypi = "line_profiler/line_profiler-2.0.tar.gz"
 
     license("PSF-2.0")
 
+    version("4.1.2", sha256="aa56578b0ff5a756fe180b3fda7bd67c27bbd478b3d0124612d8cf00e4a21df2")
+    version("3.5.1", sha256="77400208bfbd5d4341938a9a3a4fb5194f5af7fc23b2d496c913755f8310e8b8")
     version("2.1.2", sha256="efa66e9e3045aa7cb1dd4bf0106e07dec9f80bc781a993fbaf8162a36c20af5c")
     version("2.0", sha256="739f8ad0e4bcd0cb82e99afc09e00a0351234f6b3f0b1f7f0090a8a2fbbf8381")
 
+    # see pyproject.toml
     depends_on("python@2.5:", type=("build", "run"))
```

Comment:
Versions `@:3` don't build with `python@3.11:` (from trying to build, not from theoretical sources):

```suggestion
    depends_on("python@2.5:", type=("build", "run"))
    depends_on("python@:3.10", type=("build", "run"), when="@:3")
```